backend/fms_core/models/_constants.py

```diff
@@ -11,3 +12,11 @@
 INDEX_READ_FORWARD = "FORWARD"
 INDEX_READ_REVERSE = "REVERSE"
 INDEX_READ_DIRECTIONS_CHOICES = ((INDEX_READ_FORWARD, INDEX_READ_FORWARD), (INDEX_READ_REVERSE, INDEX_READ_REVERSE))
+RELEASE_FLAG = 1
```

Comment:
RELEASE_FLAG and BLOCK_FLAG are not used (by chance they seem to be conspicuously similar).